local/action.yml

```diff
@@ -0,0 +1,46 @@
+name: 'Save/Load LocalStack state'
+
+inputs:
+  name:
+    description: 'Name of the artifact'
+    required: true
+    default: 'ls-state'
+  action:
+    description: 'Action to perform (save or load)'
+    required: true
+    default: 'save'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Download PR artifact
+      uses: dawidd6/action-download-artifact@v2
+      if: ${{ inputs.action == 'load' }}
+      env:
+        workflow_id: ${{ fromJSON(format('["{0}","{1}"]', env.WORKFLOW_ID, github.event.workflow_run.workflow_id))[env.WORKFLOW_ID == ''] }} 
```

Comment:
same as above - could maybe be simplified with a ternary `.. ? .. : ..` expression.

Reply:
After some digging it seems Github Actions doesn't support the mentioned ternary operators, however there is an equivalet syntax: `${{runner.os == 'linux' && 'Yes' || 'No'}}` which could be used.
So in our case this would be something like this:
`${{ env.WORKFLOW_ID == '' && env.WORKFLOW_ID || github.event.workflow_run.workflow_id }}` 🤔

Reply:
Awesome, nice find! 👍  Thanks for sharing your insights here..